activesupport/lib/active_support/cache/redis_cache_store.rb

```diff
@@ -319,6 +319,11 @@ def clear(options = nil)
         end
       end
 
+      # Get info from redis servers.
+      def info
```

Comment:
Let's keep the name consistent and call this method `stats`.

Reply:
@rafaelfranca - done.